add.py

```diff
@@ -0,0 +1,4 @@
+# a function to add anything together
+def adding(a,b):
+	"""Add two objects together"""
+	return(a+b)
```

Comment:
Please use 4 spaces instead of this large tab (it seems to be 8 spaces).
```suggestion
    """Add two objects together"""
    return(a+b)
```